src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -114,6 +114,21 @@ public CoreNode CreateStratisPosNode(Network network)
             return CreateNode(new StratisBitcoinPosRunner(this.GetNextDataFolderName(), network), "stratis.conf");
         }
 
+        public CoreNode CreateStratisCustomPosNode(Network network, NodeConfigParameters configParameters)
```

Comment:
This appears that it will only be used by your test, so move this to the test class and just call CreateCustomNnode from there 👍